docs/stencil-docs/configure-store-design-ui/store-design-overview.md

```diff
@@ -3,55 +3,90 @@
+        "label": "i18n.BannerBackground",
+        "id": "color-primary"
+      }
+    ]
+  }
+]
+```
 
-* Use caution in deleting any key. Doing so can break your new theme version's backward compatibility.
-* We do not recommend renaming keys. Instead, we recommend introducing a new key, while maintaining the old key until it is no longer in use by anyone using an older version of your theme.
-* Each object within your <span class="fn">config.json</span> › `variations` object defines one theme variation. If you are adapting an existing theme and consciously want to remove one or more variations, you can do so by removing the corresponding key(s).
+Then, set it's default value in [`config.json`](https://github.com/bigcommerce/cornerstone/blob/master/config.json).
 
-## Persistent Settings Storage
+```json
+{
+    "settings": {
+        "color-primary": "#ff957f"
+    }
+}
 
-When store administrators use Store Design to customize your theme for their store, the store's resulting configuration settings are saved to a separate configuration service at BigCommerce.
+```
 
-## Theme Upgrades and Settings
+[Learn more about defining UI options](https://developer.bigcommerce.com/stencil-docs/configure-store-design-ui/defining-ui-options).
 
-When a merchant upgrades your theme to a newer version, all key-value pairs that were saved to the BigCommerce configuration service are carried forward. For example, assume this customization/upgrade scenario:
+## Managing keys between versions
 
-You release your Star Glow theme, version 1. This theme's <span class="fn">config.json</span> includes a key named `logo_size`, establishing a default value of 100x250. The combination of the key and the value compose a `logo_size` setting.
-The merchant uses Store Design to change the `logo_size` setting to 175x275. This customized setting is stored in the BigCommerce configuration service.
-You release Star Glow, version 1.1. In this theme revision, you have changed the `logo_size` to 300x300.
-When the merchant applies Star Glow version 1.1 to their store, their custom `logo_size` setting of 175x275 remains in effect.
-If you the merchant creates a second store and applies Star Glow version 1.1 to it, that store has no custom `logo_size` setting – so it will default to the new theme version's 300x300 value.
+We recommend developer do the following to ensure changes to theme settings are backward compatible:
+* Use caution when removing setting definitions in `schema.json`; doing so can break the theme's backward compatibility.
+* Avoid renaming settings in `schema.json`; add a new setting instead.
+
+## Persistent settings storage
+
+BigCommerce's configuration service stores setting values configured in the UI whenever a user saves and uses those values when rendering the theme on the storefront.
+
+## Theme upgrades and settings
+
+Existing settings saved to the configuration service persist across theme version updates. Here's an example scenaio:
+

```

Comment:
Image dimensions do not need  code formatting; should include the unit of measurement for clarity
300 x 300px